docs/modules/release-notes/pages/all-releases/4_13_3S.adoc

```diff
@@ -0,0 +1,107 @@
+= Kobiton 4.13.3S release notes
+:navtitle: Kobiton 4.13.3S release notes
+
+_July 19, 2024_
+
+== New OS support
+
+kobiton 4.13.3S add support for the following:
+
+* iOS/iPadOS 18 Beta
+* Xcode 16 Beta
+
+== CoreDevice: Lightning mode support for iOS 17.4 devices and later without networking changes

```

Comment:
Please take out CoreDevice: from the title of this section. Start with "Lightning mode...."

Reply:
Removed